### PR TITLE
source-greythr contribution from bhushan-dhwaniris

### PR DESCRIPTION
**Pull request by bhushan-barbuddhe**
## What

This PR adds a new connector for source GreytHr (source-greythr).

Contributor-provided connector description:

The GreytHR Connector for Airbyte allows seamless integration with the GreytHR platform, enabling users to automate the extraction and synchronization of employee management and payroll data into their preferred destinations for reporting, analytics, or further processing.
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `base_url` | `string` | Base URL. https://api.greythr.com |  |
| `domain` | `string` | Host URL. Your GreytHR Host URL |  |
| `username` | `string` | Username.  |  |
| `password` | `string` | Password.  |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| Employees | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employees Categories | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employees Profile | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employees Personal Details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employees Work Details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employee Separation Details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employee Statutory Details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employee Bank Details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employee PF &amp; ESI details | employeeId | DefaultPaginator | ✅ |  ❌  |
| Employee Qualifications Details |  | DefaultPaginator | ✅ |  ❌  |
| Users List |  | DefaultPaginator | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Can you upload an square svg icon for GreytH?

File: airbyte-integrations/connectors/source-greythr/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,1041 @@
...
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - gthost
+      - username
+    properties:
+      gthost:
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-greythr/manifest.yaml:474]:
```suggestion
      domain:
```

File: airbyte-integrations/connectors/source-greythr/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "api.greythr.com"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-greythr/metadata.yaml:5]:
```suggestion
      - "*"
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@bhushan-dhwaniris can you upload the square icon? and request my review again thanks

**Comment by bhushan-dhwaniris** [comment]
> Can you upload an square svg icon for GreytH?

@marcosmarxm done.

**Pull request closed without merging**